_fr/tour/extractor-objects.md

```diff
@@ -3,10 +3,64 @@ layout: tour
 title: Extractor Objects
 partof: scala-tour
 
-num: 14
+num: 18
 
 language: fr
 
-next-page: generic-classes
+next-page: for-comprehensions
 previous-page: regular-expression-patterns
 ---
+
+Un objet extracteur est un objet avec une méthode `unapply`. Tandis que la méthode `apply` ressemble à un constructeur qui prend des arguments et crée un objet, `unapply` prend un object et essaye de retourner ses arguments. Il est utilisé le plus souvent en filtrage par motif ou avec les fonctions partielles.
```

Comment:
```suggestion
Un objet extracteur est un objet avec une méthode `unapply`. Tandis que la méthode `apply` ressemble à un constructeur qui prend des arguments et crée un objet, `unapply` prend un object et essaye de retourner ses arguments. Il est utilisé le plus souvent en filtrage par motif (*pattern matching*) ou avec les fonctions partielles.
```
I did not know the French term "filtrage par motif". I always use "pattern matching" even when I speak in French. I wouldn't be surprised if I'm not alone. So I suggest to at least once write the English name.